.rubocop.yml

```diff
@@ -58,6 +59,16 @@ IdentityIdp/ErrorsAddLinter:
   Exclude:
     - 'spec/**/*.rb'
 
+IdentityIdp/ImageSizeLinter:
+  Enabled: true
+  Exclude:
+    - app/components/barcode_component.html.erb
+    - app/views/partials/multi_factor_authentication/_mfa_selection.html.erb
+    - app/views/shared/_nav_branded.html.erb
+    - app/views/sign_up/completions/show.html.erb
+    - app/views/users/two_factor_authentication_setup/index.html.erb
+    - app/views/users/webauthn_setup/new.html.erb
```

Comment:
Noting that I'd hoped to have disabled these rules inline at the specific point in the file where the `image_tag` is called, but it doesn't currently appear to be possible to do with ERBLint.

Upstream issue: https://github.com/Shopify/erb-lint/issues/243